BestPractices.md

```diff
@@ -80,8 +80,53 @@ This chapter outlines best practices to use for developing in iOS and Android.
 
 ## Android
 
-### Common libraries
+### Architectures
+
+### Development Advices
 
-#### - Networking - Parsing
-#### - Image Download
-#### - Dependecy Injection
+* Try not to use nested fragments in activities.
```

Comment:
I don't agree with this. What is the source of that statement?

Reply:
Using fragment inside of fragments can make it hard to access UI elements in the activity. So, if title or anything else changes in the root activity, at least 2 listener needs to be added to each activity and fragments.

Reply:
and also, I don't know if it's fixed or not but may face with the bug in the link http://delyan.me/android-s-matryoshka-problem/